velox/vector/BaseVector.cpp

```diff
@@ -357,13 +353,27 @@ void BaseVector::addNulls(const uint64_t* bits, const SelectivityVector& rows) {
       });
 }
 
-void BaseVector::clearNulls(const SelectivityVector& rows) {
+void BaseVector::addNulls(const SelectivityVector& nullRows) {
+  if (!nullRows.hasSelections()) {
+    return;
+  }
+  VELOX_CHECK(isNullsWritable());
+  VELOX_CHECK(length_ >= nullRows.end());
```

Comment:
Just a little bit of inconsistency here, for the clearnulls we dont care if nullRows.end > length_ , but for addNulls we ensure that length >= . I know it makes more sense to be strict in this case, but just pointing it out.